securedrop_client/app.py

```diff
@@ -169,7 +170,7 @@ def prevent_second_instance(app: QApplication, unique_name: str) -> None:
     except OSError as e:
         if e.errno == ALREADY_BOUND_ERRNO:
             err_dialog = QMessageBox()
-            err_dialog.setText(app.applicationName() + " is already running.")
+            err_dialog.setText(app.applicationName() + _(" is already running."))
```

Comment:
Making this something like `setText(_("{application_name} is already running".format(app.applicationName)))` would produce a message that would be easier to translate.

Reply:
we also have suggested to developers to use f-strings

Reply:
that's a good point. the `semgrep` patterns will need to be adjusted to cover those.

Reply:
@rmol, re: https://github.com/freedomofpress/securedrop-client/pull/1272#discussion_r682079513:  Thanks for catching this—amended.

@creviera, re: f-strings:  I'm under the impression that the f-string interpolation will always preempt the `gettext()` call, which is the opposite order of operations we want—at least without some [fancy footwork](https://bugs.python.org/issue36310#msg348417) with a code smell I don't love.  (Cf. also python-babel/babel#594.)  Is this the approach we want to take?

Reply:
@cfm I think you're right, I'm having trouble thinking of a case where f-strings make sense in a translation context. It's more likely they'd just be used to interpolate already-translated strings into another, not be translated themselves, so wouldn't be something we want to catch with semgrep. (And you made me realize my example above would have the same problem -- one of the parentheses at the end should instead close the `_()` call  before `.format`.)

Reply:
Very nice. I'm happy with this. @creviera ?

Reply:
:100: like the checks and the reasoning makes sense